src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Shape {
+    private double top;
+    private double bottom;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double top, double bottom, double height) {
+        super(Colors.valueOf(color));
```

Comment:
The `Colors.valueOf(color)` assumes the color string is always a valid enum constant. Consider adding error handling to manage cases where the color might not be valid.